### PR TITLE
Use ip module to get local ip address

### PR DESCRIPTION
**Pull request by fjvallarino**
@tejohnso please review

We will have to update both rise-launcher-electron and rise-cache-v2 to use this version, otherwise the build process keeps getting rise-cache-v2's version.

### CODE REVIEW

**Comment by tejohnso** [comment]
:+1:

**Comment by fjvallarino** [comment]
@tejohnso updated

File: network.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ var platform = require("./platform.js"),
 dns = require("dns"),
 fetch = require("node-fetch"),
 http = require("http"),
+nodeIp = require("ip"),
```
Comment (fjvallarino) [inline network.js]: @tejohnso removed

**Comment by tejohnso** [comment]
:+1:

**Pull request merged**